tests/e2e/vmlistener_test.go

```diff
@@ -120,7 +124,7 @@ func (s *VMListenerTestParams) TestKillVM(c *C) {
 
 	// power on vm
 	govc.PowerOnVM(s.dockerHostName)
-	misc.SleepForSec(35)
+	misc.SleepForSec(45)
```

Comment:
In addition to above comment we definitely need #1294 (not to log.fatal()) as the powerOn status comes back immediately meanwhile guest os has to bootup > NIC has to retrieve IP address until this step "host is not reachable" error hits. 

#1294 is the must need to bypass `log.fatal()`, introduced this static wait time. 

@shaominchen has raised a PR to introduce common poller #1301